src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    String getRandomColor() {
+        int colorIndex = new Random().nextInt(Color.values().length);
+        Color value = Color.values()[colorIndex];
+        return value.toString();
```

Comment:
Use name() instead of toString() for enums
```suggestion
        return value.name();
```